app/assets/javascripts/views/batch_actions_view.js.coffee

```diff
@@ -0,0 +1,143 @@
+#// Models the batch actions done on index pages
+class ELMO.Views.BatchActionsView extends ELMO.Views.ApplicationView
+
+  el: '#index_table'
+
+  events:
+    'click #select_all_link': 'select_all_clicked'
+    'click a.batch_op_link': 'submit_batch'
+    'click a.select_all_rows': 'select_all_rows'
+    'change input[type=checkbox].batch_op': 'checkbox_changed'
+    'mouseover table.index_table tbody tr': 'highlight_partner_row'
+    'mouseout table.index_table tbody tr': 'unhighlight_partner_row'
+
+  initialize: (params, search_form_view) ->
+    @is_search = params.is_search
+    @form = this.$el.find('form').first() || this.$el.closest('form')
+    @select_all_rows_field = this.$el.find('input[name=select_all]')
+    @alert = this.$el.find('div.alert')
+    @pages = this.$el.data('pages')
+    @entries = this.$el.data('entries')
+    @select_all_page = false
+    @class_name = I18n.t("activerecord.models.#{params.class_name}.many")
+    @search_form_view = search_form_view
```

Comment:
Are all of these variables still used? When you are refactoring by splitting like this it's important to check if instance vars are all necessary. I think @is_search can be deleted.